src/observations/trend_observation_percentage_array_stock.py

```
@@ -0,0 +1,37 @@
+# Extending trend_observation
+from .trend_observation import TrendObservation, TrendEnvironment
+from src.interfaces.order_interface import OrderAction
```

Ruff

`src.interfaces.order_interface.OrderAction` imported but unused

Remove unused import: `src.interfaces.order_interface.OrderAction`

(F401)